storage/range_test.go

```diff
@@ -2384,32 +2384,38 @@ func TestRangeDanglingMetaIntent(t *testing.T) {
 	}
 }
 
-// TestInternalLookupFirstRange tests that InternalRangeLookup works correctly
-// with the first range (StartKey==KeyMin). Normally we look up this range
-// in gossip instead of executing the RPC, but InternalRangeLookup is still
-// used when up-to-date information is required.
-func TestInternalRangeLookupFirstRange(t *testing.T) {
+// TestInternalLookup tests that InternalRangeLookup works correctly.
```

Comment:
s/TestInternalLookup/TestInternalRangeLookup/ but you might just lose the comment if that's all you wanna say.